eessi-2023.06-eb-4.8.0-2021b.yml

```
@@ -0,0 +1,4 @@
+easyconfigs:
+  - TensorFlow-2.7.1-foss-2021b.eb:
+      options:
+        include-easyblocks-from-pr: 2218
```

Let's add a comment to clarify why the PR is required:

```suggestion
  - TensorFlow-2.7.1-foss-2021b.eb:
      # correctly determine path to active binutils in TensorFlow easyblock,
      # see https://github.com/easybuilders/easybuild-easyblocks/pull/2218
      options:
        include-easyblocks-from-pr: 2218